spring-kafka/src/main/java/org/springframework/kafka/retrytopic/ListenerContainerFactoryConfigurer.java

```diff
@@ -95,20 +100,31 @@
 		this.clock = clock;
 	}
 
+	@Deprecated
```

Comment:
We should add javadoc to describe why they are deprecated; we usually say `@deprecated in favor of #someOtherMethod`